dp-hello-world-controller/config/config_test.go

```diff
@@ -18,9 +20,11 @@ func TestConfig(t *testing.T) {
 			})
 
 			Convey("Then the values should be set to the expected defaults", func() {
+				So(cfg.BindAddr, ShouldEqual, ":8124")
 				So(cfg.GracefulShutdownTimeout, ShouldEqual, 5*time.Second)
 				So(cfg.HealthCheckInterval, ShouldEqual, 30*time.Second)
 				So(cfg.HealthCheckCriticalTimeout, ShouldEqual, 90*time.Second)
+				So(cfg.HelloWorldEmphasise, ShouldEqual, true)
```

Comment:
This configuration value will need to be removed after initialisation of new repo - is it worth adding a comment against the assertion? Like so:

```suggestion
				So(cfg.HelloWorldEmphasise, ShouldEqual, true) // TODO Remove assertion once configuration variable has been removed from config.go
```